python/py-graph-tool/Portfile

```diff
@@ -73,11 +80,16 @@ if {${name} ne ${subport}} {
 
     # PYTHON_EXTRA_LDFLAGS is set to work around incorrect detection of
     # link flags by configure
-    configure.env-append PYTHON=${python.bin} \
-                         PYTHON_VERSION=${python.branch} \
-                         PYTHON_CPPFLAGS=-I${python.include} \
-                         "PYTHON_LDFLAGS=-L${python.libdir}/.. -lpython${python.branch}" \
-                         "PYTHON_EXTRA_LDFLAGS=-L${python.libdir}/.. -lpython${python.branch}"
+    configure.env-append \
+                    PYTHON=${python.bin} \
+                    PYTHON_VERSION=${python.branch} \
+                    PYTHON_CPPFLAGS=-I${python.include} \
+                    "PYTHON_LDFLAGS=-L${python.libdir}/.. -lpython${python.branch}" \
+                    "PYTHON_EXTRA_LDFLAGS=-L${python.libdir}/.. -lpython${python.branch}"
+
+    # https://trac.macports.org/ticket/61583#comment:8
+    macosx_deployment_target
```

Comment:
@kencu questions in the mentioned Trac ticket whether this is a proper fix to put into MacPorts. How did you try and address his concern/question?

Reply:
Yes. Upstream disagrees: https://git.skewed.de/count0/graph-tool/-/issues/702#note_3858

Reply:
1. Upstream can be wrong
2. Do we then have any idea what DEPLOYMENT_TARGET then gets baked into the objects?

I doubt anyone upstream (or here so far) has the slightest idea why setting the DEPLOYMENT_TARGET (which is the proper thing to do, as we all know) is causing this to error out, and what exactly clearing the DEPLOYEMENT_TARGET (which is the wrong thing to do, as we all know) does to fix it.

This kind of random hacking fix might get the build done. It will almost inevitably just come back to bite you on the a** again later.

Reply:
Noted!